amplipi/utils.py

```diff
@@ -133,6 +134,19 @@ def available_outputs():
     print('WARNING: ch0, ch1, ch2, ch3 audio devices not found. Is this running on an AmpliPi?')
   return outputs
 
+def configure_inputs():
+  """ The IEC598 and Aux inputs are being muted/misconfigured during system startup
+
+    Enable the capture capability for Aux and IEC958 and set capture volume to 100% (unmuted) """
+  if is_amplipi():
+    # setup usb soundcard input volumes and unmute
+    try:
+      subprocess.run(shlex.split(r'amixer -D hw:cmedia8chint set Speaker 100% unmute'), check=True) # is this required??
```

Comment:
I don't think I've ever seen the outputs get set to anything other than 100% by default, but I have seen them muted. Not sure if that was a fluke. We'll always want this state though, so I see no reason not to have this here to prevent any future problems.

Also, any reason this is a raw string?

Reply:
The sequence after the % was being highlighted in vscode, so I figured it would avoid any weird escape sequence caused by the %